server/apps/search_providers/aap_mm/__init__.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8; -*-
+#
+# This file is part of Superdesk.
+#
+# Copyright 2013, 2014 Sourcefabric z.u. and contributors.
+#
+# For the full copyright and license information, please see the
+# AUTHORS and LICENSE files distributed with this source code, or
+# at https://www.sourcefabric.org/superdesk/license
+
+from apps.search_providers import register_search_provider
+from superdesk import intrinsic_privilege
+
+PROVIDER_NAME = 'aapmm'
+
+
+def init_app(app):
+    # Must be imported here as the constant PROVIDER_NAME is referenced by the below modules
+    from .aap_mm_datalayer import AAPMMDatalayer
+    from .resource import AAPMMResource
+    from .service import AAPMMService
+
+    app.data.aapmm = AAPMMDatalayer(app)
+
+    aapmm_service = AAPMMService(datasource=None, backend=app.data.aapmm)
```

Comment:
just wondering if there should be service/resource for every search provider we have, some `search_provider_search` resource that would accept query + search provider id and delegate that to those custom datalayers (or there could be different abstraction - with only query, find_one and fetch methods maybe). to create service and resource for every provider seems like quite some overhead, and there is not much aap specific in it.